sample/src/main/java/comfolioreader/android/sample/HomeActivity.java

```diff
@@ -50,6 +65,50 @@ public void onClick(View v) {
         });
     }
 
+    private List<HighlightImpl> getHighlights() {
```

Comment:
Developer will not aware of HighlighImpl class.. It is used for internal purpose.. It will never be exposed to developer.. Create separate class which implements Highlight interface for sample project.. Say HighlighData..